### PR TITLE
06-12 release notes for workers

### PR DESCRIPTION
**Pull request by garrettgu10**

### CODE REVIEW

File: data/changelogs/workers.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ productLink: "/workers/"
 productArea: Developer platform
 productAreaLink: /workers/platform/changelog/platform/
 entries:
+- publish_date: '2024-06-12'
+  description: |-
+    - Blob and Body objects now include a new `bytes()` method.
```
Comment (irvinebroque) [inline data/changelogs/workers.yaml]: can we clarify that this is part of standard and link to it?

Comment (garrettgu10) [reply data/changelogs/workers.yaml]: I don't think this is standard? Maybe @jasnell knows more about this

Comment (irvinebroque) [reply data/changelogs/workers.yaml]: if not standard then we need link and docs

Comment (KianNH) [reply data/changelogs/workers.yaml]: Seems to be a recent addition to the two standards: https://github.com/whatwg/fetch/commit/1085f4fbfeebda8d4fa5efcb6c5c7350953e524b & https://github.com/w3c/FileAPI/commit/85b18bc93d934bfafe0c46ad285adc457f5017df

**Review by ObsidianMinor — APPROVED** [review]

**Review by irvinebroque — APPROVED** [review]

**Pull request merged**